components/cosmetic_filters/resources/data/content_cosmetic.ts

```diff
@@ -344,26 +363,26 @@ const unhideSelectors = (selectors: Set<string>) => {
   }
   // Find selectors we have a rule index for
   const rulesToRemove = Array.from(selectors)
-    .map(selector => allSelectorsToRules.get(selector))
+    .map(selector => window.content_cosmetic.allSelectorsToRules.get(selector))
```

Comment:
no need to add `window.content_cosmetic.` prefix to all of these, see previous comment